mockup/patterns/moment/pattern.js

```diff
@@ -86,7 +86,42 @@ define([
   'mockup-i18n',
   'moment'
 ], function($, Base, i18n, moment) {
-  lazyLoadMomentLocale();
+  var currentLanguage = (new i18n()).currentLanguage;
+
+  // From https://github.com/moment/moment/blob/3147fbc/src/test/moment/format.js#L463-L468
+  var MOMENT_LOCALES =
+    'ar-sa ar-tn ar az be bg bn bo br bs ca cs cv cy da de-at de dv el ' +
+    'en-au en-ca en-gb en-ie en-nz eo es et eu fa fi fo fr-ca fr-ch fr fy ' +
+    'gd gl he hi hr hu hy-am id is it ja jv ka kk km ko lb lo lt lv me mk ml ' +
+    'mr ms-my ms my nb ne nl nn pl pt-br pt ro ru se si sk sl sq sr-cyrl ' +
+    'sr sv sw ta te th tl-ph tlh tr tzl tzm-latn tzm uk uz vi zh-cn zh-tw';
+
+  lazyLoadMomentLocale(currentLanguage);
+
+  function isLangSupported(lang) {
+    return MOMENT_LOCALES.split(' ').indexOf(lang) !== -1;
+  }
+
+  function lazyLoadMomentLocale(lang) {
+    var LANG_FALLBACK = 'en';
```

Comment:
There is something fishy in this function.
You pass the lang variable and then reevaluate it (twice).
By just reading it I feel puzzled by the code.
Anyway not a merge blocker, we can improve this later.

Reply:
I'll see what I can improve. I put 2 early exit conditions because I didn't want to apply transforms and checks to the original `lang` string if it's `en`, it's minor but I tried to avoid because this code is currently executed at every page load.

Reply:
I don't pass `lang` anymore but I'm not sure how to improve more. Now that I don't reassign a received param, do you think it's fine to reassign it twice?

I could replace the ternary by plain ifs but I found it more verbose and less logical as the two first parts of the ternary repeat.

Reply:
If that is causing much pain than value ignore my comment. 
It is not a merge blocker and it was just suggesting bad feelings :)
Really if this is ready please merge it!

Reply:
Your opinion is very welcome so I'm glad to remove code smells! I appreciate it!

Reply:
just noticed var LANG_FALLBACK should live outside function. If there are any other pending changes I improve that :)